spec/dependent_validations_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+require '../lib/mutations/dependent_validations'
+
+describe "Mutations::DependentValidations" do
+
+  describe "invalid graphs" do
+    class CyclicCommand
```

Comment:
This seems to match how some of the other specs I looked at in this repo are written, but I'm unfamiliar with a) quite a lot of the code in this repo and b) minitest. Happy to rewrite if you want.